webhook_router/webhook/parse.py

```diff
@@ -0,0 +1,111 @@
+#  Copyright 2024 Canonical Ltd.
+#  See LICENSE file for licensing details.
+
+"""Module for parsing the webhook payload."""
+
+from webhook_router.webhook import Job
+
+WORKFLOW_JOB = "workflow_job"
+Flavor = str
+Label = str
```

Comment:
I like the strong type hinting :D 
It doesn't seem like it's being used anywhere though?
There's also a duplicate definitions for these in `webhook_router/webhook/label_translation.py`

Reply:
Yes, this now lives in `label_translation` and was forgotten to be removed here during the refactoring. Thank you very much.